### ISSUE TITLE
Kmail message viewer prohibits cut/copy

### ISSUE THREAD

**Issue by adamf663** [k9mail/k-9#3368]
### Expected behavior
Long press on message window selects text.

### Actual behavior
Cut and copy disabled

### Steps to reproduce
1.view message
2.long press anywhere in message
3.

### Environment
K-9 Mail version:

Android version:

Account type (IMAP, POP3, WebDAV/Exchange):

**Comment by cketti** [comment]
We use a system component for displaying the message contents. If the copy functionality is not showing for you that's most likely an issue with your OS, not K-9 Mail.

**Comment by adamf663** [comment]
No other email client has this issue.  I have to his reply to gain cut and copy capability so it is clearly not an Oscar issue.

**Comment by adamf663** [comment]
No other email client has this issue.  I have to his reply to gain cut and copy capability so it is clearly not an Oscar issue.

**Comment by adamf663** [comment]
Please reopen. 

On May 1, 2018 12:52:29 PM MDT, cketti <notifications@github.com> wrote:
>We use a system component for displaying the message contents. If the
>copy functionality is not showing for you that's most likely an issue
>with your OS, not K-9 Mail.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/k9mail/k-9/issues/3368#issuecomment-385755729

-- 
Sent from my Android phone with K-9 Mail. Please excuse my brevity.

**Comment by cketti** [comment]
We're not doing anything to deliberately disable the copy functionality. It's working on all of my devices. You didn't post any information about which K-9 Mail version you're using, what device you're using, or what OS version. If you have any concrete information on what could be causing this we'll be happy to include a workaround. But I don't see any of the existing developers spending time on trying to find out why it's not working for you.

**Comment by adamf663** [comment]
Sorry about the bother.
I just tested again and it worked.  Not sure what was different.
FYI:  kmail 5.403, android oreo 8.1, on a pixel 2.


----- Original Message -----

**Comment by CStinnett69** [comment]
Hi there. Forgive me if it seems I was making accusations. There is a line above my browser search line that says "Kmail message viewer prohibits cut/copy. Is..." It won't show and I can't get to the rest of it. I tried copying and pasting the link and it.        G gives error "404 Record not found ". The files I was trying to send were screenshots in .png format. At the top of my screen there is a message "Switch to Mobile version which is what I'm on. That's what I meant about this problem. They are on a PC emulating my Android. I downloaded another app called IP Connection List giving me all kinds of info. There is an export option, I'm going to try that.  Thanks guys. I'll send it under separate cover.
